lib/lita/handlers/totems.rb

```diff
@@ -14,7 +14,7 @@ class Totems < Handler
         opal:"#pd-contributor-demos",
         vert:"#pd-contributor-demos",
         cyan:"#demo-deployment-cyan",
-        plum:"#squad-integrity",
+        plum:"#squad-site-platform-qa",
```

Comment:
we could use this opportunity to rename our qa channel to `demo-deployment-plum` ?

cc @tomoyaogura

Reply:
This would make it easier to route crashloops on plum environment to that channel/ integration test failures/etc.

Reply:
Created corresponding PR in B/G deployment side, not that there's much there https://github.com/change/devops_chef/pull/4263
For EKS I think we'll need to add logic to actually [send notifs](https://github.com/change/jenkins-infra/blob/master/jenkinsfiles/terraform-deploy.groovy#L167).

Although another  point to standardize is `#pd-teal-deploys` vs `#pd-deployment-cyan` although I think `#pd-deployment-<demo>` is better in terms of how it sorts alphabetically.